package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "server": "nodemon index.js",
     "start": "node index.js",
-    "test": "cross-env NODE_ENV=testing jest --watch --verbose",
+    "test": "cross-env NODE_ENV=test jest --watch --verbose",
```

Comment:
Why are we changing this?

Reply:
VSCodes debugger was setting it to `test`, so tests checking for correct environment would fail, because they expected `testing`, but got `test`.